repo/packages/O/omsagent/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+ "packagingVersion": "2.0",
```

Comment:
Renamed the package to msoms so that it will be consistent all over the places.
Changed packagingVersion to 3.0

minDcosReleaseVersion 
   We are talking to our partners to figure out exact minimum version, we will add it later.